packages/mdx/lib/plugin/recma-jsx-rewrite.js

```diff
@@ -136,11 +136,25 @@ export function recmaJsxRewrite(options = {}) {
             const fullId = ids.join('.')
             const id = name.name
 
+            const isInScope = inScope(currentScope, id)
+
             if (!own.call(fnScope.references, fullId)) {
-              fnScope.references[fullId] = {node, component: true}
+              const parentScope = /** @type {Scope|null} */ (
```

Comment:
is this cast needed? What else can it be?

Reply:
It's needed because this `Scope` type is defined locally

https://github.com/mdx-js/mdx/blob/0df684bc343a4b89ecf0a849fe8fefe52233386b/packages/mdx/lib/plugin/recma-jsx-rewrite.js#L19

Without this cast the type is `periscopic.Scope` and `node` is not there.